rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/filter/TbOriginatorTypeFilterNodeConfiguration.java

```diff
@@ -31,7 +31,7 @@
     public TbOriginatorTypeFilterNodeConfiguration defaultConfiguration() {
         TbOriginatorTypeFilterNodeConfiguration configuration = new TbOriginatorTypeFilterNodeConfiguration();
         configuration.setOriginatorTypes(Arrays.asList(
-                EntityType.DEVICE
+                EntityType.ALARM
```

Comment:
No need to change the default configuration. Otherwise - LGTM. Please correct this and I will merge.

Reply:
It is done.